src/main/java/barcus/ui/Ui.java

```diff
@@ -0,0 +1,59 @@
+package barcus.ui;
+
+import java.util.Scanner;
+
+/**
+ * Class to handle displaying to the user and accepting user input
+ */
+public class Ui {
+
+    private Scanner scanner;
+
+    /**
+     * Constructor for Ui object, creates a scanner
+     */
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Prints message s with appropriate string in front
+     * @param s
+     */
+    public void talk(String s) {
+        System.out.println("Barcus: " + s);
+    }
+
+    /**
+     * Prints welcome message
+     */
+    public void showWelcome() {
+        talk("Beep bop! Hello I am Barcus, ready to be of assistance!\n"
```

Comment:
Like how you stored all the constant information into UI and appending to the string as needed depending on the command.